pages/Variations/[imageID].js

```diff
@@ -0,0 +1,70 @@
+import Image from "next/image.js";
+import styled from "styled-components";
+import { StyledA } from "../../components/StyledLink";
+import useSWR from "swr";
+import { useRouter } from "next/router.js";
+
+export default function ChooseFourVariations() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { imageID } = router.query;
+
+  const { data, isLoading, error } = useSWR(`/Variations/${imageID}`);
+
+  if (!isReady || isLoading || error) return <h2>Loading...</h2>;
```

Comment:
do you actually need `useSWR` here?

We use useSWR to fetch external data (from an API), you don't appear to be doing this just yet.

You can remove those lines of code